src/main/java/build/buildfarm/instance/shard/JedisClusterFactory.java

```diff
@@ -290,4 +310,70 @@ private static Set<HostAndPort> list2Set(String[] nodes) throws ConfigurationExc
       throw new ConfigurationException(e.getMessage());
     }
   }
+
+  /**
+   * @brief Attach ssl factory to builder.
+   * @details Create an SSL context and attach it to the builder.
+   * @param builder
+   * @return A builder with the SSL context attached.
+   * @note Suggested return identifier: builder.
+   */
+  private static DefaultJedisClientConfig.Builder attachSslSocketFactoryIfNeeded(
```

Comment:
can we break this out into a separate RedisSSL.java - it pollutes the namespace of this file substantially and has only static methods to show for it - I'm picturing one road in with the cert file, and one road out with the authentication adapter.

Reply:
Sure - shall I place it under `src/main/java/build/buildfarm/common/redis/RedisSSL.java` ?

Reply:
Yup, that will work best.